Makefile

```diff
@@ -64,14 +65,42 @@ rpm:
 	packit build locally
 	mv $(shell uname -m)/flightctl-*.rpm bin/rpm
 
+
+# cross-building for deb pkg
+bin/amd64:
+	GOARCH=amd64 go build -buildvcs=false $(GO_BUILD_FLAGS) -o $@/flightctl ./cmd/flightctl/...
+	GOARCH=amd64 go build -buildvcs=false $(GO_BUILD_FLAGS) -o $@/flightctl-agent ./cmd/flightctl-agent/...
```

Comment:
do we want to include the server or is that separate?

Reply:
The idea is that server would be provided as container, this is only to be installed in systems that we want to manage.